packages/block-library/src/image/view.js

```diff
@@ -77,269 +77,278 @@ function handleScroll( ctx ) {
+								ctx.lastFocusableElement
+						) {
+							event.preventDefault();
+							ctx.firstFocusableElement.focus();
+						}
+					}
 
-				if ( event.key === 'Escape' || event.keyCode === 27 ) {
-					actions.hideLightbox( event );
+					if ( event.key === 'Escape' || event.keyCode === 27 ) {
+						actions.hideLightbox( event );
+					}
 				}
-			}
-		},
-		// This is fired just by lazily loaded
-		// images on the page, not all images.
-		handleLoad() {
-			const ctx = getContext();
-			const { ref } = getElement();
-			ctx.imageLoaded = true;
-			ctx.imageCurrentSrc = ref.currentSrc;
-			callbacks.setButtonStyles();
-		},
-		handleTouchStart() {
-			isTouching = true;
-		},
-		handleTouchMove( event ) {
-			const ctx = getContext();
-			// On mobile devices, we want to prevent triggering the
-			// scroll event because otherwise the page jumps around as
-			// we reset the scroll position. This also means that closing
-			// the lightbox requires that a user perform a simple tap. This
-			// may be changed in the future if we find a better alternative
-			// to override or reset the scroll position during swipe actions.
-			if ( ctx.lightboxEnabled ) {
-				event.preventDefault();
-			}
-		},
-		handleTouchEnd() {
-			// We need to wait a few milliseconds before resetting
-			// to ensure that pinch to zoom works consistently
-			// on mobile devices when the lightbox is open.
-			lastTouchTime = Date.now();
-			isTouching = false;
-		},
-	},
-	callbacks: {
-		initOriginImage() {
-			const ctx = getContext();
-			const { ref } = getElement();
-			ctx.imageRef = ref;
-			if ( ref.complete ) {
+			},
+			// This is fired just by lazily loaded
+			// images on the page, not all images.
+			handleLoad() {
+				const ctx = getContext();
+				const { ref } = getElement();
 				ctx.imageLoaded = true;
 				ctx.imageCurrentSrc = ref.currentSrc;
-			}
-		},
-		initTriggerButton() {
-			const ctx = getContext();
-			const { ref } = getElement();
-			ctx.lightboxTriggerRef = ref;
-		},
-		initLightbox() {
-			const ctx = getContext();
-			const { ref } = getElement();
-			if ( ctx.lightboxEnabled ) {
-				const focusableElements =
-					ref.querySelectorAll( focusableSelectors );
-				ctx.firstFocusableElement = focusableElements[ 0 ];
-				ctx.lastFocusableElement =
-					focusableElements[ focusableElements.length - 1 ];
-
-				// Move focus to the dialog when opening it.
-				ref.focus();
-			}
+				callbacks.setButtonStyles();
+			},
+			handleTouchStart() {
+				isTouching = true;
+			},
+			handleTouchMove( event ) {
+				const ctx = getContext();
+				// On mobile devices, we want to prevent triggering the
+				// scroll event because otherwise the page jumps around as
+				// we reset the scroll position. This also means that closing
+				// the lightbox requires that a user perform a simple tap. This
+				// may be changed in the future if we find a better alternative
+				// to override or reset the scroll position during swipe actions.
```

Comment:
```suggestion
				/*
				 * On mobile devices, prevents triggering the
				 * scroll event because otherwise the page jumps around as
				 * the scroll position is reset. This also means that closing
				 * the lightbox requires that a user perform a simple tap. This
				 * may be changed in the future if there is a better alternative
				 * to override or reset the scroll position during swipe actions.
				 */
```